### PR TITLE
Add a setup for tests for the admin rest api

### PR DESCRIPTION
**Pull request by danyx23**
This PR adds a framework to tests our internal admin API via http fetch calls. The main purpose of the PR is to add the scaffold, the tests themselves are not yet particularly useful.

### CODE REVIEW

File: adminSiteServer/app.test.tsx [hunk 1]
```diff
@@ -1,18 +1,196 @@
...
+import fs from "fs"
 
 jest.setTimeout(10000) // wait for up to 10s for the app server to start
+let testKnexInstance: Knex<any, unknown[]> | undefined = undefined
+let serverKnexInstance: Knex<any, unknown[]> | undefined = undefined
+let app: OwidAdminApp | undefined = undefined
+let cookieId: string = ""
+
+beforeAll(async () => {
+    // jest.setup.js
```
Comment (rakyi) [inline adminSiteServer/app.test.tsx:62]: What does this comment mean? Could you expand on it?

File: adminSiteServer/app.test.tsx [hunk 2]
```diff
@@ -1,18 +1,196 @@
...
+    const _ = google.docs
```
Comment (rakyi) [inline adminSiteServer/app.test.tsx:63]: Why are we doing this?

**Review by rakyi — APPROVED** [review]
LGTM overall, except I wasn't sure why we do a few things.

**Comment by danyx23** [comment]
@rakyi thanks for you feedback and sorry it took me so long to respond! I added some comments - is that what you were looking for or is anything still unclear?

**Comment by rakyi** [comment]
LGTM!

**Pull request merged**